server/ansible/roles/pbench-repo-install/vars/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+baseurl: "{{ 'http://coprbe.devel.redhat.com/results/ndokos/pbench/rhel-8.dev-$basearch/'
```

Comment:
Can we remove the internal URL?

Reply:
Yes - in fact, this has gone away completely now.